replica/table.cc

```diff
@@ -3864,6 +3864,7 @@ future<row_locker::lock_holder> table::do_push_view_replica_updates(shared_ptr<d
     auto lock = co_await std::move(lockf);
     auto pk = dht::partition_range::make_singular(m.decorated_key());
     auto permit = co_await sem.obtain_permit(base, "push-view-updates-read-before-write", estimate_read_memory_cost(), timeout, tr_state);
+    co_await utils::get_local_injector().inject("prolong_view_reads_100ms", std::chrono::milliseconds(100));
```

Comment:
Question: when injections are disabled, is this inject and co_await thing compiled out?

I think we're adding too many injections, but don't know if I should be just displeased or actually worried ;-)

Reply:
Based on 
https://github.com/scylladb/scylladb/blob/20c2d6210e673d824fafcddd8e5d504aca8547f7/utils/error_injection.hh#L561-L563
they're supposed to be optimized away, but I suspect we're still doing a preemption check there, maybe we should start using `seastar::coroutine::without_preemption_check` with injections to avoid even that